src/journal/migrations/0049_auto_20220303_1636.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2022-03-03 21:36
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('journal', '0048_merge_20220110_1933'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='journal',
+            name='display_article_number',
+            field=models.BooleanField(default=False, help_text='Some journals give each article in a volume its own number rather than subdividing the volume into issues. If you display article numbers, it is recommended to turn off display of issue numbers. Note that article number is an optional field separate from article ID and can be set in Edit Metadata.'),
```

Comment:
Just curious about the tip `If you display article numbers, it is recommended to turn off display of issue numbers`. Is that the case? We have a number of journals that have used Volume, Issue, and article numbers in the past

Reply:
Ah interesting--in that case I will change the tip.